src/sonic-yang-models/yang-templates/sonic-types.yang.j2

```diff
@@ -205,7 +205,23 @@ module sonic-types {
         }
     }
 
-    typedef copp_packet_action {
+    typedef policer_mode {
+        type enumeration {
+            enum sr_tcm;
+            enum tr_tcm;
+            enum storm;
+        }
+    }
+
+    typedef policer_color_source {
+        type enumeration {
+	    enum aware;
+	    enum blind;
+        }
+    }
+
+
+    typedef policer_packet_action {
```

Comment:
~~Wonder why it is called "policer_packet_action". This enum derives from sai_packet_action_t and can be reused accross different modules, not just policer. I suggest calling it just "packet_action".~~

EDIT: I see that in the implementation inside orchagent, copporch and policerorch packet_action_map are two different maps, thus I suggest to leave copp_packet_action and add policer_packet_action with same content to reflect that. Otherwise adding a new action to packet_action_map inside copporch would require automatic support of this action from policerorch when adding it here, which might not be supported.

Ideally, copporch should be reusing the implementation of policerorch and then it could be a single typedef of policer_packet_action but I don't see that is how it works.

Reply:
@stepanblyschak Currently both policerorch and copporch support the same actions. IMO the code is duplicate (which should be cleaned up) and I don't see a scenario where an action supported in copp orch will not be supported in policerorch or vice versa. In fact in both examples we just map the value to SAI attribute and invoke create policer. Since copp uses policer I feel it's better not to duplicate it at yang level.